test/generators/install_generator_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+require_relative File.expand_path("../../../lib/generators/accessly/install/install_generator", __FILE__)
+
+class Accessly::InstallGeneratorTest < ::Rails::Generators::TestCase
```

Comment:
this test will create a temp directory inside the root folder of the project, run the generator, and test that the named migration exists... also tests for something namespaced that we didn't create just for sanity of nil/not nil